app/templates/view_users.html

```diff
@@ -23,87 +21,71 @@
+        "text": "Find a user by email",
+        "classes": "govuk-label--m",
+        "isPageHeading": false
+      },
+      "id": "input-email_address",
+      "name": "email_address",
+    })}}
 
     {{ govukButton({
       "text": "Search"
     }) }}
 
   </form>
 
-  {% call(item) summary.list_table(
-    users,
-    caption="Users",
-    empty_message="No users to show",
-    field_headings=[
-        'Name',
-        'Role',
-        'Supplier',
-        'Last login',
-        'Last password change',
-        'Locked',
-        summary.hidden_field_heading("Change status")
-    ],
-    field_headings_visible=True)
-  %}
-    {% call summary.row() %}
-
-      {{ summary.field_name(item.name) }}
-
-      {{ summary.text(item.role) }}
-
-      {% call summary.field() %}
-        {% if item.role == 'supplier' %}
-          <a class="govuk-link" href="{{ url_for('.find_suppliers', supplier_id=item.supplier.supplierId) }}">{{ item.supplier.name }}</a>
+  {% if users %}
+    {% set ns = namespace(rows = []) %}
+    {% for user in users %}
+      {% set supplier %}
+        {% if user.role == 'supplier' %}
+          <a class="govuk-link" href="{{ url_for('.find_suppliers', supplier_id=user.supplier.supplierId) }}">{{ user.supplier.name }}</a>
         {% endif %}
-      {% endcall %}
+      {% endset %}
 
-      {% call summary.field() %}
-        {% if item.loggedInAt %}
-          {{ item.loggedInAt|timeformat }}<br/>
-          {{ item.loggedInAt|dateformat }}
+      {% set logged_in_at %}
+        {% if user.loggedInAt %}
+          {{ user.loggedInAt|timeformat }}<br/>
+          {{ user.loggedInAt|dateformat }}
         {% else %}
           Never
         {% endif %}
-      {% endcall %}
+      {% endset %}
 
-      {% call summary.field() %}
-        {% if item.passwordChangedAt %}
-          {{ item.passwordChangedAt|timeformat }}<br/>
-          {{ item.passwordChangedAt|dateformat }}
+      {% set password_changed_at %}
+        {% if user.passwordChangedAt %}
+          {{ user.passwordChangedAt|timeformat }}<br/>
+          {{ user.passwordChangedAt|dateformat }}
         {% else %}
           Never
         {% endif %}
-      {% endcall %}
+      {% endset %}
 
-      {% call summary.field() %}
-        {% if item.locked and current_user.has_any_role('admin', 'admin-ccs-category') and not item.personalDataRemoved %}
-          <form action="{{ url_for('.unlock_user', user_id=item.id) }}" method="post">
+      {% set locked %}
+        {% if user.locked and current_user.has_any_role('admin', 'admin-ccs-category') and not user.personalDataRemoved %}
+          <form action="{{ url_for('.unlock_user', user_id=user.id) }}" method="post">
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
             <input type="hidden" name="source" value="{{ url_for('.find_user_by_email_address', email_address=email_address) }}"/>
             {{ govukButton({
               "classes": "govuk-button--secondary",
```

Comment:
I think if we add a `govuk-!-margin-bottom-0` to this, it'll align like the rest of the row. From the screenshot, it currently looks slightly off.

Reply:
Nice - looks much better now:

![image](https://user-images.githubusercontent.com/15256121/125276420-19619080-e308-11eb-909a-11b98108f500.png)